### PR TITLE
Service mgr

### PR DESCRIPTION
**Pull request by Eabramov84**

### CODE REVIEW

File: servicemanager/servicemanager.go [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+	"github.com/aoscloud/aos_servicemanager/utils/uidgidpool"
+)
+
+/***********************************************************************************************************************
+ * Consts
+ **********************************************************************************************************************/
+
+const maxConcurrentActions = 10
+
+const tmpTarFile = "tmpTar.tar.gz"
```
Comment (al1img) [inline servicemanager/servicemanager.go:45]: tmpfs.tar.gz, tmp.tar.gz

File: database/database_internal_test.go [hunk 2]
```diff
@@ -96,68 +97,60 @@ func TestMain(m *testing.M) {
...
 	}
 
-	if !reflect.DeepEqual(service, service1) {
-		t.Errorf("service1 doesn't match updated one: %v", service)
+	if services, err = db.GetAllServices(); err != nil {
+		t.Errorf("Can't get all services: %s", err)
+	}
+
+	if len(services) != 3 {
+		t.Errorf("incorrect count of all services %d", len(services))
```
Comment (al1img) [inline database/database_internal_test.go:153]: Start from capital letter.

File: database/database_internal_test.go [hunk 3]
```diff
@@ -171,219 +164,38 @@ func TestNotExistService(t *testing.T) {
...
-	if err != nil {
-		t.Errorf("Can't get services: %s", err)
+	if !reflect.DeepEqual(serviceFromDB, service) {
+		t.Error("service1 doesn't match stored one")
 	}
 
-	if len(servicesSp2) != 2 {
-		t.Error("Wrong service count")
+	if err := db.RemoveService(service); err != nil {
+		t.Error("Can't delete service: ", err)
```
Comment (al1img) [inline database/database_internal_test.go:194]: Errorf %s

File: database/database_internal_test.go [hunk 4]
```diff
@@ -171,219 +164,38 @@ func TestNotExistService(t *testing.T) {
...
 	}
 
-	for _, service := range servicesSp2 {
-		if !reflect.DeepEqual(service, service3) && !reflect.DeepEqual(service, service4) {
-			t.Error("Error getting services")
-		}
+	if _, err := db.GetService("service1"); err == nil {
+		t.Errorf("Should be error not exist ")
```
Comment (al1img) [inline database/database_internal_test.go:198]: Should be error:

File: servicemanager/servicemanager.go (hunk 1 above)
Comment (Eabramov84) [reply servicemanager/servicemanager.go:45]: renamed

File: database/database_internal_test.go (hunk 3 above)
Comment (Eabramov84) [reply database/database_internal_test.go:194]: done

File: database/database_internal_test.go (hunk 2 above)
Comment (Eabramov84) [reply database/database_internal_test.go:153]: done

**Review by al1img — APPROVED** [review]

**Review by al1img — APPROVED** [review]

**Pull request merged**